vue/sbc-common-components/src/services/fee.services.ts

```diff
@@ -22,6 +23,7 @@ export default {
       promises.push(Axios.get(url, { headers: { Authorization: `Bearer ${token}` } }))
     }
 
+    // @ts-ignore
```

Comment:
Can you look into fixing the statement instead of ignoring the warnings?

Reply:
Fixed in next commit.
The cause was that, in case of a catch, "void" was being returned. I solved this (not by updating the return type to `Promise<void | Fee[]>` but instead) by returning `[]` in case of an error. This satisfies the typing.